README.md

```diff
@@ -50,7 +50,11 @@ All three proxies respect the XML configuration properties listed below.
 * Use the ProxyConfig tag to specify the following proxy level settings.
     * **mustMatch="true"** : When true only the sites listed using serverUrl will be proxied. Set to false to proxy any site, which can be useful in testing. However, we recommend setting it to "true" for production sites.
     * **allowedReferers="http://server.com/app1,http://server.com/app2"** : A comma-separated list of referer URLs. Only requests coming from referers in the list will be proxied. See https://github.com/Esri/resource-proxy/issues/282 for detailed usage.
-    * **logFile="proxylog.txt"** : When a logFile is specified, the proxy will log messages to this file.
+    * **logFile="proxylog.txt"** : When a logFile is specified, the proxy will log messages to this file. If only a filename is supplied for the logFile, the proxy uses the folder where the proxy.config file is found. *N.B.: The folder containing the logFile must be writable by the web server.*
```

Comment:
Did you verify what the default log file location is for the Java and PHP proxies?